config.py

```diff
@@ -740,6 +740,11 @@
     ```
 """
 
+overlap: Optional[float] = None
```

Comment:
I would rename this:
```suggestion
epochs_overlap: Optional[float] = None
```
this is more explicit!